src/control/lib/control/auto_test.go

```diff
@@ -326,16 +345,18 @@ func TestControl_AutoConfig_checkNetwork(t *testing.T) {
 			expConfigOut: config.DefaultServer().WithServers(
 				ioserver.NewConfig().
 					WithFabricInterface("ib0").
+					WithFabricInterfacePort(31416).
```

Comment:
Would be better to use the default port constant that you added in these tests

Reply:
yes , will revise in future PR

Reply:
addressed in https://github.com/daos-stack/daos/pull/3820